package.json

```
@@ -38,14 +38,38 @@
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master"
   },
   "devDependencies": {
-    "jshint": "~2.10.2",
+    "eslint": "~5.16.0",
+    "eslint-config-tamia": "~6.2.1",
+    "eslint-plugin-prettier": "~3.1.0",
+    "prettier": "~1.14.2",
     "mocha": "6.1.4",
     "mongodb": "3.2.3",
     "should": "13.2.3",
-    "istanbul": "0.4.5",
+    "husky": "~1.1.0",
+    "istanbul": "^0.4.5",
```

```suggestion
    "istanbul": "0.4.5",
```

~0.4.5 also would be valid

---

Fixed: f27466e